test/test_quicktypes.jl

```diff
@@ -128,7 +128,7 @@ end
 # Another way to "support" QuickTypes with type parameters is to use
 # QuickTypes.construct.
 @qstruct_fp Plane2(nwheels, weight::Number; brand=:zoomba)
-Setfield.constructor_of(::Type{<: Plane2}) =
+Setfield.constructorof(::Type{<: Plane2}) =
```

Comment:
I suppose we need to use `ConstructionBase.constructorof` here?  Same for `test_settable.jl`.